src/docs/development/tools/vs-code.md

```diff
@@ -238,15 +238,15 @@ You can also define custom snippets by executing
 
 ### Keyboard shortcuts
 
-**Hot reload**
-: During a debug session, clicking the **Restart** button on the
-  **Debug Toolbar**, or pressing `Ctrl`+`Shift`+`F5`
-  (`Cmd`+`Shift`+`F5` on macOS) performs a hot reload.
+**Hot Reload**
```

Comment:
@aayushbisen, we use sentence case for titles and headings:  https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

Please undo the changes to the headings. thx

Reply:
Done.